go.mod

```diff
@@ -5,18 +5,18 @@ go 1.18
 require (
 	github.com/go-logr/logr v1.2.3
 	github.com/golang/glog v1.0.0
-	github.com/google/go-cmp v0.5.8
+	github.com/google/go-cmp v0.5.9
 	github.com/kennygrant/sanitize v1.2.4
-	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/gomega v1.19.0
+	github.com/onsi/ginkgo/v2 v2.5.0
```

Comment:
I'd rather have this change together with the changes in the code, to keep commits atomic and buildable